src/stripPhoneNumber/index.js

```
@@ -0,0 +1,17 @@
+function stripPhoneNumber(phoneNumber) {
+  if (typeof phoneNumber === 'string' || (phoneNumber instanceof String)) {
+    const stripedNumber = phoneNumber.replace(/\D/g, '').trim();
+    if(stripedNumber.length >= 11 || stripedNumber.length < 10) {
```

Should we throw an error if number is less than 10?